test/README.md

```diff
@@ -0,0 +1,108 @@
+# Media Insights Engine Testing - How To
+
+### Overview
```

Comment:
Minor: I think the markdown formatting in this section is not working.  I reviewed the rendered document and there was some format characters (*s) showing.

Reply:
Those are intentional. The asterisks are just there to act as a callout symbol, not markdown formatting.